functional-tests/www/index.js

```diff
@@ -20,37 +21,53 @@ nunjucks.configure(app.get('views'), {
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 
-app.get('/', function(req, res) {
+app.get('/', function (req, res) {
   res.render('index.njs');
 });
 
-app.get('/drag-and-drop', function(req, res) {
+app.get('/basic-auth', basicAuth('admin', 'password'), function (req, res) {
+  res.render('basic-auth/index.njs');
+})
+
+app.get('/recaptcha', function (req, res) {
+  res.render('form/recaptcha.html')
+});
+
+app.post('/recaptcha',function(req,res){
+  if(req.body['g-recaptcha-response'] === undefined || req.body['g-recaptcha-response'] === '' || req.body['g-recaptcha-response'] === null) {
+    return res.json({"responseCode" : 1,"responseDesc" : "Please select captcha"});
+  }
```

Comment:
const array = [undefined,null, '']
if(array.includes(req.body['g-recaptcha-response'])){
    return res.json({"responseCode" : 1,"responseDesc" : "Please select captcha"});
}

Reply:
Honestly, if you care about '' than if (array) is enough...

Reply:
well, in this case, it's enough to check just if `if (!req.body['g-recaptcha-response'])`